src/maybe/maybe.py

```diff
@@ -1,3 +1,5 @@
+# mypy: disable-error-code="import-not-found"
```

Comment:
Unfortunately, no. That's the trick I had to use to allow both linting and testing to work both with and without `result` installed. However, given your other comment about running tests _only_ with `result` installed, I might be able to move this to the specific import line. Let me get back to you after some experimentation.

Reply:
Oh, actually, since we want to assume `result` is always installed by CI and contributors, we can completely eliminate this and we also don't need any "ignore" comments in the import line itself.